js/modules.js

```diff
@@ -11,17 +11,14 @@ module.exports = (ENV, { MODULES, THIRDPARTY }) => {
   return {
     rules: [
       {
-        // .js and .jsx files are caught
-        test: /\.jsx?$/,
+        // .js, .mjs, and .jsx files are caught
+        test: /\.m?jsx?$/,
         exclude: new RegExp(`(${MODULES}|${THIRDPARTY})`),
         loader: 'babel-loader',
         options: {
           presets: [
-            ['env', { modules: false }],
-            'react',
-          ],
-          plugins: [
-            'transform-object-rest-spread',
+            ['@babel/preset-env', { useBuiltIns: 'usage', corejs: '3', modules: 'commonjs' }],
+            '@babel/preset-react',
```

Comment:
`env` and `react` are replaced with the modern `@babel/preset-env` and `@babel/preset-react` respectively. The new options are necessary, since babel's polyfills are replaced with `core-js`.

The `transform-object-rest-spread` polyfill is no longer needed, as we don't support any browsers that don't natively support the functionality it polyfills.